### ISSUE TITLE
NVDA stops speaking when certain errors are encountered on the windows logon screen in win8.1

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5232]
**Reported by Mohamed on 2015-07-18 09:56**
So, I have noticed that at times, if NVDA encounters errors on the windows logon screen in windows 8.1, it stops speaking. An example of this is if you get something like a user profile service error saying that the profile cannot be loaded. The error will appear, and, as soon as the OK button is pressed, all speech from NVDA stops.

**Comment by Mohamed00** [comment]
I am the original reporter of this issue. This problem occurred in Windows 8, and I believe it may have been before NVDA had Ease of Access integration. I don't see this issue as of the latest version of NVDA, and unless this is still happening, I believe it should be closed.

**Comment by leonardder** [comment]
Ok, thanks for coming back to this. Closing as works for me, but as @Mohamed00 mentioned, please give a shout if this is still the case for you.
